app/models/api-topics.model.js

```diff
@@ -0,0 +1,13 @@
+const db = require('../../db/connection')
+
+exports.selectTopics = () => {
+    let sqlQuery = 'SELECT * FROM topics'
+
+    sqlQuery += ';'
```

Comment:
You can put the ; in line 4 without needing to reassign 🙂

Reply:
I was thinking more if I need to add logic in the future to sort the topics but I guess I could just add it in the future if I need to